src/backend/joanie/core/filters/client.py

```diff
@@ -31,11 +33,18 @@ class OrderViewSetFilter(filters.FilterSet):
         choices=enums.PRODUCT_TYPE_CHOICES,
         exclude=True,
     )
+    query = filters.CharFilter(method="filter_by_query")
```

Comment:
Why not naming it `product_title` for clarity ?

Reply:
I implemented the name `query` to match what was done into the file `joanie.core.filter` -> `admin.py` and this [PR](https://github.com/openfun/joanie/commit/20c8465d6e71a64eded11d67b50ed6f5ca3ad5d0)

Reply:
What is your opinion on this suggestion ? @jbpenrath @lunika , We need your thoughts on this subject.

Reply:
`query` is already used as you said so IMO we should keep this logic.